xml/sec_scf_values_yaml.xml

```diff
@@ -0,0 +1,1894 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE sect1
+[
+  <!ENTITY % entities SYSTEM "entity-decl.ent">
+    %entities;
+]>
+<sect1
+ xml:id="app.scf.values.yaml" 
+ xmlns="http://docbook.org/ns/docbook"
+ xmlns:xi="http://www.w3.org/2001/XInclude"
+ xmlns:xlink="http://www.w3.org/1999/xlink"
+ version="5.0">
+
+    <title>Complete suse/scf values.yaml file</title>
+  <para>
+   This is the complete output of <command>helm inspect suse/cf</command> for
+   the current &productname; release.
```

Comment:
Would it make sense to specify _which_ release was current as of time of writing? (Even if this is auto-generated via a script at publishing time)

Reply:
yes good idea